cmd/webhook/main.go

```diff
@@ -109,8 +121,11 @@ func main() {
 		Namespace:                       system.Namespace(),
 		Port:                            8443,
 		SecretName:                      "webhook-certs",
-		WebhookName:                     "webhook.serving.knative.dev",
+		ResourceMutationWebhookName:     "resource.webhook.serving.knative.dev",
```

Comment:
It is useful to see what this looks like all together, but I think we should stage this breaking change separately from the rest to make rollbacks easier (if needed).

Reply:
it slipped through in my earlier refactoring attempt 😔 ... sure will pull this out as a separate commit.